### DISCUSSION TITLE
Avoid PHP-FPM Reloading - 'SCRIPT_FILENAME' doesn't show current path

### DISCUSSION THREAD

**Discussion by pmochine** [deployphp/deployer#2862] [Help needed]
I followed this [tutorial](https://deployer.org/docs/7.x/avoid-php-fpm-reloading), 
however when echoing `echo $_SERVER['SCRIPT_FILENAME'];`  I still get this path `/var/www/staging/releases/2/public/index.php`.

Any idea what to do? It should say `current` in the path.

My nginx file

```
    location ~ \.php$ {
        include fastcgi_params; //I have changed the order to the top as you see it in the tutorial. Laravel puts it down below. Does it even matter?
        fastcgi_pass unix:/var/run/php/php8.0-fpm.sock;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
    }
```

**Comment by antonmedv** [comment]
It should NOT include current in the path.

**Reply by pmochine** [reply]
@antonmedv ahhh. I guess I should not work if I'm tired. Thanks!
